src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -165,6 +164,127 @@ Keep in mind that Let's Encrypt may send you emails about your certificate neari
 
 - **insecure_secrets_log** <span id="insecure_secrets_log"/> enables logging of TLS secrets to a file. This is also known as `SSLKEYLOGFILE`. Uses NSS key log format, which can then be parsed by Wireshark or other tools. ⚠️ **Security Warning:** This is insecure as it allows other programs or tools to decrypt TLS connections, and therefore completely compromises security. However, this capability can be useful for debugging and troubleshooting.
 
+### Trust Pool Providers
+
+These are the standard trust pool providers that can be used in the `trust_pool` subdirective:
+
+#### inline
+
+The `inline` module parses the trusted root certificates as listed in the Caddyfile directly in base64 DER-encoded format. The `trust_der` directive may be repeated multiple times.
+
+```caddy-d
+... inline {
```

Comment:
I think we should just use `trust_pool` everywhere for this instead:

```suggestion
trust_pool inline {
```

In other places we didn't do that because where the module can be placed in config is ambiguous so what `...` can be can change, but in this case it'll always be `trust_pool` I think.